pkg/internal/runtime/runtime.go

```diff
@@ -18,6 +18,8 @@ func GetDefault(logger log.Logger) cluster.ProviderOption {
 	case "docker":
 		logger.Warn("using docker due to KIND_EXPERIMENTAL_PROVIDER")
 		return cluster.ProviderWithDocker()
+	case "nerdctl", "finch":
+		return cluster.ProviderWithNerdctl(p)
```

Comment:
Would it be possible to support `nerdctl.lima` too?

Reply:
Yes!

Reply:
Done